ecosystem.js

```diff
@@ -3,7 +3,7 @@ export { default as debug } from "./es/can-debug";
 export { default as defineBackup } from "./es/can-define-backup";
 export { default as defineStream } from "./es/can-define-stream";
 export { default as defineStreamKefir } from "./es/can-define-stream-kefir";
-export { default as kefir } from "./es/can-kefir";
+export { default as Kefir } from "./es/can-kefir";
```

Comment:
This is a breaking change. We should export both for now.

Reply:
Fixed

Reply:
Does this work?

```js
export { default as kefir, default as Kefir } from "./es/can-kefir";
```

Reply:
I'll check it.